arcgis-ios-sdk-samples/Maps/Map loaded/MapLoadedViewController.swift

```diff
@@ -35,17 +35,18 @@ class MapLoadedViewController: UIViewController {
         self.mapView.map = self.map
         
         //register as an observer for loadStatus property on map
-        self.map.addObserver(self, forKeyPath: "loadStatus", options: .new, context: nil)
+        self.map?.addObserver(self, forKeyPath: "loadStatus", options: .new, context: nil)
     }
     
     override func observeValue(forKeyPath keyPath: String?, of object: Any?, change: [NSKeyValueChangeKey : Any]?, context: UnsafeMutableRawPointer?) {
         
         //update the banner label on main thread
         DispatchQueue.main.async { [weak self] in
             
-            if let weakSelf = self {
-                //get the string for load status
-                let loadStatusString = weakSelf.loadStatusString(weakSelf.map.loadStatus)
+            //get the string for load status
+            if let weakSelf = self, let loadStatus = weakSelf.map?.loadStatus {
```

Comment:
This should be `strongSelf`. Ryan has created issue runtime/cocoa-common-sdk#913